Sources/Mini/Utils/Extensions/ObservableType+Extensions.swift

```diff
@@ -0,0 +1,72 @@
+/*
+ Copyright [2019] [BQ]
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import Foundation
+import Combine
+
+@available(iOS 13.0, *)
+public extension Publisher {
+
+    func filterKey(_ keyPath: KeyPath<Output, Bool>) -> AnyPublisher<Output, Failure> {
```

Comment:
`Publishers.Filter<Self>`?

Reply:
This extension of operators I have been very close to delete it all, since in Combine none of them are needed, it has its own native operators. But I understand that for use in Rx it may still make sense to keep them, but they are irrelevant for us, they would not be needed. If you want me to adapt or change any of this, let me know exactly what.

Reply:
The return type of the function, wouldn't be more correct to be my suggestion?

Reply:
Fixed!